Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM python:3.10
+FROM python:3.10 AS base
```

Comment:
**thought**: Can this run on python 3.12 or 3.13 which are still getting support?

https://devguide.python.org/versions/

Reply:
I don't remember the context of this project's python version number. I created a new issue for upgrading it.